tests/page/page-network-request.spec.ts

```diff
@@ -20,7 +20,7 @@ import { attachFrame } from '../config/utils';
 import fs from 'fs';
 
 function adjustServerHeaders(headers: Object, browserName: string) {
-  if (browserName === 'firefox')
+  if (browserName === 'firefox' || browserName === '_bidiFirefox')
     delete headers['priority'];
```

Comment:
Can this header be properly reported in `network.beforeRequestSent` instead? This is a workaround of the playwright limitation. Probably we can avoid the same bug in the Bidi instrumentation?

Reply:
FYI the `priority` header got [implemented in Firefox 128](https://bugzilla.mozilla.org/show_bug.cgi?id=1865040) which was released about a year ago. 

Also [we don't have `priority` as property for the base parameters for network events in WebDriver BiDi](https://w3c.github.io/webdriver-bidi/#cddl-type-networkbaseparameters). @hbenl lets file an issue for the BiDi spec - and maybe you want to get it added?

Until it's in the spec and landed in Firefox lets keep the test failing but tag it accordingly in the Playwright status spreadsheet. Thanks.

Reply:
The above comment is not fully correct. The priority header is actually part of the `request`, which is already included. So there is no change specifically for the WebDriver BiDi spec needed.